### PR TITLE
sync form item

### LINKED ISSUE #14772: form组件 高级搜索例子 显示错误提示时在ie所有浏览器下布局错乱
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://ant.design/components/form-cn/#components-form-demo-advanced-search](https://ant.design/components/form-cn/#components-form-demo-advanced-search)

### Steps to reproduce
在输入框输入一个值，然后按退格键删除

### What is expected?
布局正常
image

### What is actually happening?
布局错误
image

| Environment | Info |
|---|---|
| antd | 3.13.2 |
| React | react16 |
| System | win10 |
| Browser | ie11 |

### PR DESCRIPTION
**Pull request by zombieJ**
First of all, thank you for your contribution! 😄

New feature please send pull request to feature branch, and rest to master branch.
Pull request will be merged after one of collaborators approve.
Please makes sure that these form are filled before submitting your pull request, thank you!

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### What's the background?

fix #14772

### What's the effect? (Optional if not new feature)

Add placeholder element to occupy the position.

### Changelog description (Optional if not new feature)

> 1. Fix FormItem not align when has error message.
> 2. 修复 FormItem 在有错误提示时的布局问题。

### Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by zombieJ** [comment]
@afc163 , pls help to review: https://deploy-preview-14778--ant-design.netlify.com/components/form-cn/#header

File: components/form/FormItem.tsx [hunk 1]
```diff
@@ -157,23 +157,29 @@ export default class FormItem extends React.Component<FormItemProps, any> {
...
-        transitionAppear
-        key="help"
-        onEnd={this.onHelpAnimEnd}
-      >
-        {children}
-      </Animate>
+      <div className={`${prefixCls}-explain-holder`}>
+        <span aria-hidden="true" className={`${prefixCls}-explain-misc`}>
+          {'\u00A0'}
+        </span>
```
Comment (afc163) [inline components/form/FormItem.tsx]: 直接用伪元素是不是更简单。

**Comment by zombieJ** [comment]
Updated. Pls check.

**Review by afc163 — APPROVED** [review]

**Comment by coldice945** [comment]
@zombieJ  @afc163 
ant-form-explain-holder 会一直都有。无论该 form item 有没有提示或者 rules；
可以看看官网 [form-cn](https://ant.design/components/form-cn/)，3.13.3 的版本和 3.13.2 的版本样式有变化。比如 
```
        <Form.Item
          wrapperCol={{ span: 12, offset: 5 }}
        >
          <Button type="primary" htmlType="submit">
            Submit
          </Button>
        </Form.Item>
```
下面也空了一大部分。
这块代码变动这么大，建议多测测。。。

File: components/form/FormItem.tsx [hunk 2]
```diff
@@ -157,23 +157,26 @@ export default class FormItem extends React.Component<FormItemProps, any> {
...
-      <Animate
-        transitionName="show-help"
-        component=""
-        transitionAppear
-        key="help"
-        onEnd={this.onHelpAnimEnd}
-      >
-        {children}
-      </Animate>
+      <div className={`${prefixCls}-explain-holder`}>
```
Comment (sosohime) [inline components/form/FormItem.tsx]: 这个holder现在一直显示导致布局乱了

Comment (zombieJ) [reply components/form/FormItem.tsx]: > 这个holder现在一直显示导致布局乱了

看看这个 PR 解决问题不？ #14886

Comment (sosohime) [reply components/form/FormItem.tsx]: > > 这个holder现在一直显示导致布局乱了
> 
> 看看这个 PR 解决问题不？ #14886

解决了，感觉这种影响基础布局的改动还是要在谨慎点

Comment (afc163) [reply components/form/FormItem.tsx]: @sosohime 感谢鞭策，我们的锅。

**Pull request merged**